asyncio_glib/glib_selector.py

```diff
@@ -7,27 +7,17 @@
     'GLibSelector',
 )
 
-
-# The override for GLib.MainLoop.run installs a signal wakeup fd,
-# which interferes with asyncio signal handlers.  Try to get the
-# direct version.
-try:
-    g_main_loop_run = super(GLib.MainLoop, GLib.MainLoop).run
-except AttributeError:
-    g_main_loop_run = GLib.MainLoop.run
-
-
 class _SelectorSource(GLib.Source):
     """A GLib source that gathers selector """
 
-    def __init__(self, main_loop):
+    def __init__(self, selector):
         super().__init__()
         self._fd_to_tag = {}
         self._fd_to_events = {}
-        self._main_loop = main_loop
+        self._iteration_timeout = 0
```

Comment:
Nitpick, I would drop the leading '_' or set it through a function, honoring the python convention of 'private variables'.

Reply:
I think it is completely normal to add a leading `_` for internal (module private) variables. Or is that for private to subclasses only?

Reply:
Not explicitly defined in PEP8 however as in lots of (if not all) languages where there exists a strict private/protected keyword, this applies to the context of definition. Thus in this case the `_SelectorSource` class, which on its turn can be and is marked as internal within the module scope (which will exclude it and all its members from a default import). 
On line 83 and 85 the `_iteration_timeout` variable is accessed from outside of the class scope, hence implying it should actually be public.
But again, it's just a nitpick (and probably something contested, which you get with conventions)